internal/config/load.go

```diff
@@ -66,6 +68,16 @@ func Load(workingDir, dataDir string, debug bool) (*Config, error) {
 		cfg.Options.Debug,
 	)
 
+	if bts, err := exec.CommandContext(context.Background(), "git", "rev-parse", "--is-inside-work-tree").CombinedOutput(); err != nil || strings.TrimSpace(string(bts)) != "true" {
+		const depth = 2
```

Comment:
```suggestion
		const depth = 4
```

Reply:
i found out that 2 works better, because otherwise it can easily use all the max items on a single folder